indico/modules/events/registration/controllers/management/registration_tags.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from flask import flash, redirect, request, session
+from webargs import fields
+
+from indico.core.db import db
+from indico.modules.events.registration import logger
+from indico.modules.events.registration.controllers.management import RHManageRegFormsBase
+from indico.modules.events.registration.controllers.management.reglists import RHRegistrationsActionBase
+from indico.modules.events.registration.forms import RegistrationTagForm, RegistrationTagsAssignForm
+from indico.modules.events.registration.models.registrations import RegistrationTag
+from indico.modules.events.registration.views import WPManageRegistration
+from indico.util.i18n import _
+from indico.web.args import use_kwargs
+from indico.web.flask.util import url_for
+from indico.web.util import jsonify_data, jsonify_template
+
+
+class RHManageRegistrationTags(RHManageRegFormsBase):
+    """List all registration tags for an event."""
+
+    def _process(self):
+        tags = self.event.registration_tags.all()
+        tags.sort(key=lambda tag: str.casefold(tag.name))
```

Comment:
If you change the lazy=dynamic from my other comment, replace this with `tags = sorted(self.event.registration_tags, ...)` or maybe even move the sorting into the jinja template (`|sort(attribute='name')`) - then you can even use `event.registration_tags` directly in the template